hack/ci/run-conformance-tester.sh

```diff
@@ -27,11 +27,6 @@ export VERSIONS=${VERSIONS_TO_TEST:-"v1.18.8"}
 export EXCLUDE_DISTRIBUTIONS="${EXCLUDE_DISTRIBUTIONS:-ubuntu,centos,sles,rhel}"
 export ONLY_TEST_CREATION=${ONLY_TEST_CREATION:-false}
 
-export WORKER_NAME="${BUILD_ID}"
-if [ "${KUBERMATIC_NO_WORKER_NAME:-}" = "true" ]; then
-  WORKER_NAME=""
-fi
-
```

Comment:
Since we test in kind, there are never worker-names involved. This is a leftover from when we tested in a shared master/seed cluster.